### PR TITLE
100.6.0

### PR DESCRIPTION
**Pull request by tschie**
updated samples and new samples for version 100.6.0

- [x] Will update version numbers this Friday.
- [x] Before merging this PR, verify that HTML pages generated from `100.6.0` branch look and behave correctly.

### CODE REVIEW

**Comment by JonLavi** [comment]
Perhaps we could gitignore the `gradle-wrapper.jar` files, since they are  generated for each sample when you run `./gradlew run`?

**Comment by markjdugger** [comment]
@tschie @Rachael-E @JonLavi @mbcoder @colinanderson 
Please verify the generated HTML at following link:
https://mark9880-javasampdirectpublish-preview-dev.developers.arcgis.com/java/latest/java/sample-code/

**_This should be a prerequisite before merging `100.6.0` to `master`._**

WARNING: Make sure you at at the path above (/java/latest/java/sample-code).  If you click `Sample Code` on nav2 or the Sample Code tile on the Runtime SDK for Java home page, you will be taken to the **_wrong_** path:  `/java/latest;/sample-code/`

For the moment, both 100.5 and 100.6 versions of Sample Code are being generated. I am working with Dev Experience to stop building the 100.5 version.

**Comment by JonLavi** [comment]
@markjdugger thanks for the link! It's very exciting to have this auto-generated.

On first glance I can spot the following:
- The page title is `JavaSamples`, missing a space between the words `<h1 class="trailer-1">JavaSamples</h1>`
- there's a MD style box under the title (`<pre>` tag), following 'You have some choices for getting started [...]' which seems to have too much indentation hence causing it to be a code block in MD, and breaks the links.
- On some of the 'Sample Code' snippet windows (HTML tag: `<code class="java hljs">`), when the code has very long lines, the user can scroll very far to the right. Are we ok with this behaviour or should/can we put in artificial line wrapping? (perhaps by setting a max width to the `<code>` element? Haven't really worked with those in HTML.) Screenshot of what I am seeing: image
- On the above, these `<code>` tags create a very 'tall' page, where we can scroll down for a long while. Do we want to give those a set height, and scroll within them? We'd reach the `<footer>` element quicker and it would make selecting a different sample in the menu on the left a lot quicker, not having to scroll up from the very bottom of the page.
- Should we have a linke somewhere on each page to return to the main samples page? (https://mark9880-javasampdirectpublish-preview-dev.developers.arcgis.com/java/latest/java/sample-code/). Once we go on an individual sample's details, it feels like we can't naturally navigate back to this page.

**Comment by tschie** [comment]
@markjdugger @JonLavi Can we move this discussion to Slack please?

**Comment by tschie** [comment]
> Perhaps we could gitignore the gradle-wrapper.jar files, since they are generated for each sample when you run ./gradlew run?

I think `gradlew` wouldn't work without the jar file. Perhaps IntelliJ is generating/downloading it for you. Not sure that would work from the command line/terminal.

I'm not too worried about including them because they are mostly all the same and I think git would deduplicate them.

**Comment by JonLavi** [comment]
Just noticed that 'eclipse_proj.png' and 'intellij_proj.png' aren't representative any more following the split structure.

File: README.md [hunk 1]
```diff
@@ -1,80 +1,67 @@
...
-## Importing into an IDE
-We will step through how to import the Samples project into Eclipse and IntelliJ IDEA. Other IDEs may support Gradle too. Please consult their documentation for importing Gradle projects.
-
-### IntelliJ IDEA
-After cloning the samples, open IntelliJ IDEA and follow these steps:
-
-1. Click *Import Projects* from the Welcome Screen or select *File > New > Project from Existing Sources*.
-2. In the select path dialog, select the `build.gradle` file in the `arcgis-runtime-samples-java` directory. Click *OK* after specifying the `build.gradle` file.
-3. Click *OK* at the next dialog to complete the import.
+first time you call the `run` task. To refresh the data, you can run the `clean` and `unpackData` tasks.
```
Comment (Rachael-E) [inline README.md]: minor thing, looks like the continuation of this sentence has jumped onto a new line

**Review by Rachael-E — APPROVED** [review]
just noticed a small formatting change to be made, other than that looks good I think. @tschie , we were discussing adding in the Javadoc to each build.gradle. I think it would make sense to add that in now, what do you think?

```
plugins {
    id 'application'
    id 'org.openjfx.javafxplugin' version '0.0.5'
    id 'idea'
    id 'eclipse'
}

idea.module.downloadJavadoc = true
eclipse.classpath.downloadJavadoc = true
```

**Comment by tschie** [comment]
@Rachael-E This close to the release, I'd prefer not to add those plugins. We can certainly do it right after the release though.

**Pull request merged**